scripts/create_initial_image_index/README.md

```
@@ -0,0 +1,263 @@
+```
+
+## ログファイル
+
+スクリプト実行時、ログは以下の両方に出力されます：
+
+1. **コンソール**: 実行中の進捗をリアルタイムで確認
+2. **ログファイル**: 実行履歴を保存（デフォルト: `logs/create_initial_index_YYYYMMDD_HHMMSS.log`）
+
+### ログファイル名の形式
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)